akka-http-core/src/main/java/akka/http/javadsl/model/headers/HttpChallenge.java

```diff
@@ -17,22 +17,6 @@
 
     public abstract Map<String, String> getParams();
 
-    /**
-     * @deprecated Use constructor with optional realm parameter instead.
-     */
-    @Deprecated
-    public static HttpChallenge create(String scheme, String realm) {
```

Comment:
Do we know what's the reasoning behind deprecating those? They seem like useful auxiliary constructors.

Reply:
The idea was to "move to Paths everywhere", but I don't mind keeping them